speech/auto_punctuation/auto_punctuation_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestAutoPunctuation(t *testing.T) {
+	testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	err := autoPunctuation(&buf, "../testdata/commercial_mono.wav")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !strings.Contains(buf.String(), "Okay. Sure.") {
```

Comment:
Should print the transcript that was returned. See https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures.

```go
if got := buf.String(); !strings.Contains(got, "Okay. Sure.") {
	t.Fatalf(`autoPunctuation(../testdata/commercial_mono.wav) = %q; want "Okay. Sure"`, got)
}
```

Reply:
Got it, thanks.